templates/assignee_no_login_nag.html

```diff
@@ -0,0 +1,25 @@
+<p>
+    The following {{ plural('bug assigned to you has', data, pword='bugs assigned to you have') }}bugs assigned to you have been unassigned due to inactivity:
```

Comment:
This email will also be sent to the triage owners and managers, so "bug assigned to you" does not apply to all cases.

Reply:
Fixed in 272324ea800dd1fd0e41c3fafe1102ad7a041cb6!

Reply:
We could have it a bit more general. We could use something like "The follwoing bugs have been unassigned due to assignee inactivity".

Reply:
Done in 4150b84c393d63b4fe97760d81928ab6a8891174!